ebnf/syntax/ebnf.sdf3

```diff
@@ -11,9 +11,16 @@ context-free start-symbols
 context-free syntax
   
   Start.Modules = <<{Module "\n"}+>>
-  Start.Grammar = <<{GrammarDef "\n"}+>>
+  Start.Grammar = <<StartSymbols?> <{GrammarDef "\n"}+>>
   
-  Module.Module = <Module <ModuleName> <{GrammarDef "\n"}+>>
+  Module.Module = <module <ModuleName>
+  									<StartSymbols?>
+  									<{GrammarDef "\n"}+>
```

Comment:
Should we introduce a separation between start symbols and the grammardef if there are start symbols? Maybe some separation would be nice.